index.html

```diff
@@ -185,5 +185,38 @@ <h1>Skills</h1>
                  </ul>
              </div>
           </section>
+          <footer class="footer-items">
+             <div class="feedback">
+              <p class="feedback-text">
+                  I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+              </p>                    
+              <form action="https://formspree.io/your-email" method="POST">
+                  <input class="form-input" type="text" id="fullname" name="fullname" placeholder="Full name" maxlength="30" required>
+                  <input class="form-input" type="email" id="email" name="email" placeholder="Email address" required>
+                  <textarea id="message" name="message" rows="5" placeholder="Write me something..." maxlength="500" required></textarea>                        
+                  <input class="submit-btn" type="submit" value="Get in touch">
+              </form>
+            </div>
+            <hr class="footer-break">
+          
+              <ul class="contact-links">
+                <li class="icons">
+                    <a href="https://github.com/Obimbo07" ><img src="img/github.svg" alt="GitHub" /></a>
+                </li>
+                <li class="icons">
+                    <a href="https://www.linkedin.com/in/austin-obimbo-9a613623a/"  ><img src="img/Linkedin icon.svg" alt="LinkedIn" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img src="img/angel list.svg" alt="angel" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img src="img/twitter.svg" alt="twitter" /></a>
+                </li>
+                <li class="icons">
+                    <a href=""><img src="img/medium.svg" alt="Medium" /></a>
+                </li>
+              </ul>
```

Comment:
- [ ] Well done for using a list to display the social icons! However, the visual appearance of the icons doesn't match the design specifications. Please make sure to update the icon colors to match the design. This will help maintain consistency and ensure that the icons align with the desired aesthetic. Keep up the great work! 👍🌟

![image](https://github.com/Obimbo07/Portfolio-2/assets/55840999/3fc4a0dc-1a99-4cc5-8b88-266eb154ed75)